### PR TITLE
Appdrusti 18

### PR DESCRIPTION
**Pull request by BOSSHK-SPEC**

### CODE REVIEW

File: lib/BottomNavigation.dart [hunk 1]
```diff
@@ -6,49 +6,104 @@ import 'package:flutter/material.dart';
...
+      ),
+    ));
   }
-  static  List<Widget> _widgetOptions = <Widget>[
+
+  static List<Widget> _widgetOptions = <Widget>[
     Home(),
     Announcement(),
-    Text('Search Page', style: TextStyle(fontSize: 35, fontWeight: FontWeight.bold)),
+    Text('Search Page',
```
Comment (sahana-drusti) [inline lib/BottomNavigation.dart]: change the text/check can we remove it

File: lib/BottomNavigation.dart [hunk 2]
```diff
@@ -6,49 +6,104 @@ import 'package:flutter/material.dart';
...
+                      child: Center(
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: <Widget>[
+                            Text('Create',style: TextStyle(color: Colors.black, fontSize: 20),),
+
+                            TextButton.icon(
+                              icon: Icon(Icons.add_circle_outline_sharp,size: 40,color: Colors.black,),
+                              label:Text('Create student',style: TextStyle(color: Colors.black, fontSize: 20),),
+                              onPressed: (){},
```
Comment (sahana-drusti) [inline lib/BottomNavigation.dart]: add redirection as we already have page ready

File: lib/HomePage.dart [hunk 3]
```diff
@@ -267,13 +148,14 @@ class HomeState extends State {
...
 
   void checkChoiceAndRedirect(String choice) {
-    switch(choice){
-      case "Profile":{
-        Navigator.push(
-          context,
-          MaterialPageRoute(builder: (context) => MyProfile()),
-        );
-      }
+    switch (choice) {
```
Comment (sahana-drusti) [inline lib/HomePage.dart]: remove this method

File: pubspec.yaml [hunk 4]
```diff
@@ -31,6 +31,8 @@ dependencies:
   email_validator: ^2.0.1
   http: ^0.13.3
   shared_preferences: ^2.0.6
+  custom_pop_up_menu: ^1.2.0
```
Comment (sahana-drusti) [inline pubspec.yaml]: remove unnecessary dependencies

File: lib/BottomNavigation.dart (hunk 1 above)
Comment (BOSSHK-SPEC) [reply lib/BottomNavigation.dart]: removed

File: lib/HomePage.dart (hunk 3 above)
Comment (BOSSHK-SPEC) [reply lib/HomePage.dart]: ok

File: pubspec.yaml (hunk 4 above)
Comment (BOSSHK-SPEC) [reply pubspec.yaml]: ok

File: lib/BottomNavigation.dart (hunk 2 above)
Comment (BOSSHK-SPEC) [reply lib/BottomNavigation.dart]: done

**Review by sahana-drusti — APPROVED** [review]

**Pull request merged**